src/broker/cas_execute.c

```diff
@@ -11753,4 +11765,36 @@ ux_cgw_free_stmt (T_SRV_HANDLE * srv_handle)
 {
   cgw_free_stmt (srv_handle);
 }
+
+static char
+ux_cgw_get_stmt_type (char *stmt)
+{
+  char stmt_type = CUBRID_STMT_NONE;
+  const char *comment_start = strstr (stmt, "/*");
+  if (comment_start)
```

Comment:
If hints other than the STMT type hint could be sent to the gateway, hint parsing will have to be repeated. If STMT type hint is only one, its ok.

Reply:
The current format has a hint at the beginning of sql, so only the first hint was searched. ( /* DBLINK SELECT */ select * from t1; )
I will consider the case of multiple hints.